modules/clientutils.js

```diff
@@ -356,9 +423,11 @@
                 var pSelector = this.processSelector(selector);
                 if (pSelector.type === 'xpath') {
                     return this.getElementByXPath(pSelector.path, scope);
-                } else {
-                    return scope.querySelector(pSelector.path);
                 }
+                if (pSelector.type === 'link text') {
```

Comment:
2 votes for "link_text" (paazmaya+mickaelandrieu)
1 vote for "link text"
0 vote for "linkText" 

:+1: